bump_version.sh

```diff
@@ -27,6 +29,10 @@ else
       tmp_file=/tmp/version.$$
       sed "s/$old_version_regex/$new_version/" $VERSION_FILE > $tmp_file
       mv $tmp_file $VERSION_FILE
+      sed "s/$old_version_regex/$new_version/" > $FLAKE_FILE
```

Comment:
This line isn't actually doing anything.  You need to write to a temp file and copy it over, like we do for the version file.

(FWIW, we can't use `sed -i` here because OSX's version of `sed` doesn't provide that functionality. :disappointed:)

Reply:
@jsf9k Please see https://github.com/cisagov/awssh/pull/27/commits/a86f0f5b217669e26ff32153fcf815ebd27c918c

Reply:
Resolved with a86f0f5 and 54df713.